chainerx_cc/chainerx/routines/activation.cc

```diff
@@ -0,0 +1,21 @@
+
+#include <vector>
+
+#include "chainerx/array.h"
+#include "chainerx/constant.h"
+#include "chainerx/device.h"
+#include "chainerx/dtype.h"
+#include "chainerx/kernels/math.h"
+#include "chainerx/routines/activation.h"
+#include "chainerx/routines/creation.h"
+#include "chainerx/routines/explog.h"
+#include "chainerx/routines/indexing.h"
+#include "chainerx/scalar.h"
```

Comment:
* Move ` #include "chainerx/routines/activation.h" ` to the first line of the file.
* It seems some includes are not necessary (constant.h, dtype.h, kernels/math.h, <vector>)